config/initializers/time_and_date_formats.rb

```diff
@@ -1,6 +1,9 @@
 Time::DATE_FORMATS[:mon_year] = "%b-%Y"
 Date::DATE_FORMATS[:mon_year] = "%b-%Y"
 
+Date::DATE_FORMATS[:mon_year_multiline] = lambda { |date| date.strftime("%b-%Y").tr("-", "\n") }
+Time::DATE_FORMATS[:mon_year_multiline] = lambda { |date| date.strftime("%b-%Y").tr("-", "\n") }
```

Comment:
Silly question, can we not put the newline char directly into a string literal?

`Date::DATE_FORMATS[:mon_year_multiline] = "%b\n%Y"?